guides/source/action_mailer_basics.md

```diff
@@ -204,9 +204,12 @@ class UsersController < ApplicationController
 end
 ```
 
-NOTE: Active Job's default behavior is to execute jobs ':inline'. So, you can use
-`deliver_later` now to send emails, and when you later decide to start sending
-them from a background job, you'll only need to set up Active Job to use a queueing
+NOTE: Active Job's default behavior is to execute jobs `:async`. So, you can use
+`deliver_later` now to send emails asynchronously.
+Active Job's default adapter runs jobs with an in-process thread pool.
+It's well-suited for dev/test since it doesn't need an external infrastructure,
```

Comment:
for the development/test environments, since it doesn't require any external infrastructure